packages/driver/cypress/integration/commands/querying_spec.js

```diff
@@ -1573,13 +1573,13 @@ describe('src/cy/commands/querying', () => {
     it('will not find script elements', () => {
       cy.$$('<script>// some-script-content </script>').appendTo(cy.$$('body'))
 
-      cy.contains('some-script-content').should('not.match', 'script')
+      cy.contains('some-script-content').should('not.exist')
```

Comment:
This code was added at #5079. It had to be written in this way because `cy.contains()` returns `<body>` tag at that time. 

It is changed, because it simply fails.